CONTRIBUTING.md

```diff
@@ -39,6 +39,53 @@ Anything contained within the _src_ folder is meant to be merged from the upstre
 
 There is no intention to support any mkdocs extensions in this theme because this theme is designed for the Sphinx documentation generator. You will often find that most of the extensions available for mkdocs are natively implemented in Sphinx or available in the form of a Sphinx extension.
 
+## Building this project's docs
+
+An example of instructions for Ubuntu 24.04:
+
+```
+$ lsb_release -a
+Distributor ID:  Ubuntu
+Description:     Ubuntu 24.04.3 LTS
+Release:         24.04
+Codename:        noble
+```
+
+Fetch source code:
+
+```
+$ git clone git@github.com:jbms/sphinx-immaterial
+$ cd sphinx-immaterial/
+```
+
+Install uv package manager and Graphviz (for dot):
+
+```
+$ sudo apt update
+$ sudo apt install snapd
+$ sudo snap install astral-uv --classic
+$ sudo apt install graphviz
+```
+
+Run build:
+
+```
+$ uv sync
+$ uvx nox -s build
```

Comment:
1. `uv sync` is the only required command here.
2. This command should be described as "Setup Python virtual environment"
3. Please remove the leading `$` from every command in these code blocks.
   - These commands are not specific to using a bash shell
   - It is very annoying to click the "copy this code" button and then paste the leading `$` into my terminal. Even if I was using bash, the leading `$` makes the copied command improper bash script syntax (not executable).
   - Seriously, this is a _very_ annoying practice, and I keep seeing it more and more in technical docs nowadays.